var/spack/repos/builtin/packages/py-fenics-ufl/package.py

```diff
@@ -34,4 +37,7 @@ class PyFenicsUfl(PythonPackage):
     version("2016.2.0", tag="ufl-2016.2.0")
 
     depends_on("python@3.7:", type=("build", "run"))
-    depends_on("py-numpy", type=("build", "run"))
+    depends_on("py-setuptools@58:", type=("build", "run"), when="@2022.1.0:")
+    depends_on("py-setuptools@40:", type=("build", "run"))
+    depends_on("py-numpy@1.21:", type=("build", "run"), when="@2022.1.0:")
+    depends_on("py-numpy@1.15:", type=("build", "run"))
```

Comment:
Is the numpy min version documented anywhere?

Reply:
My mistake. 1.21 is the min version. Versions lower do not work with py3.7+